packages/blade/jest.native.config.js

```diff
@@ -0,0 +1,24 @@
+const ignores = ['/node_modules/'];
+
+module.exports = {
+  preset: 'react-native',
+  testPathIgnorePatterns: [...ignores, 'web.test'],
+  collectCoverageFrom: ['./src/**/*.native.{ts,tsx}'],
+  coverageThreshold: {
+    global: {
+      branches: 75,
+      functions: 75,
+      lines: 75,
+      statements: 75,
+    },
+  },
+  moduleFileExtensions: ['native.ts', 'native.tsx', 'ts', 'tsx', 'js', 'json', 'node'],
+  testMatch: ['**/__tests__/**/*.native.test.{ts,tsx}', '**/__tests__/**/*.test.{ts,tsx}'],
```

Comment:
testing all the files that matches
`.native.test.ts|js|tsx|jsx` and platform independednt files `.test.ts|js|tsx|jsx`